packages/wonder-blocks-tooltip/components/tooltip-anchor.js

```diff
@@ -69,15 +82,15 @@ export default class TooltipAnchor extends React.Component<Props, State> {
             );
         }
 
+        this._unsubscribeFromTracker = TRACKER.subscribe(this);
         this._anchorNode = anchorNode;
         this._updateFocusivity();
         if (anchorNode) {
-            // TODO(somewhatabstract): Detect ESC key to deactivate
             /**
              * TODO(somewhatabstract): Work out how to allow pointer to go over
              * the tooltip content to keep it active. This likely requires
              * pointer events but that would break the obscurement checks we do.
-             * So, careful consideration required.
+             * So, careful consideration required. See
              */
```

Comment:
See...?

Reply:
Good catch! I raised an issue for this and need to add in the issue number. :grin: